Papr/Scenes/Search/SearchViewController.swift

```diff
@@ -66,10 +66,25 @@ class SearchViewController: UIViewController, BindableType {
             .disposed(by: disposeBag)
 
         tableView.rx.itemSelected
+            .map({ [unowned self] indexpath -> IndexPath? in
```

Comment:
We can make it simpler. 
Instead of `.map { ... }` we can use ` .flatMapIgnore { ... }` 

Like: 

```swift 
 tableView.rx.itemSelected
    .flatMapIgnore { [unowned self] _ in
        Observable.just(self.searchBar.endEditing(true))
     }
    .map { $0.row }
    .bind(to: inputs.searchTrigger)
    .disposed(by: disposeBag)
```

Reply:
ok, I had hard time yesterday with this, trying to make it work to close the keyboard before push to new screen (with your idea it pushes the screen same time it closes the keyboard)